src/Components/App/App.js

```diff
@@ -2,17 +2,21 @@ import React from 'react';
 import { useState, useEffect } from 'react';
 import { Switch, Route } from 'react-router-dom';
 import Books from '../Books/Books';
+import BookInfo from '../BookInfo/BookInfo';
 import Header from '../Header/Header';
 import './App.css';
 import sampleBooks from '../../sampleData/sampleBooks';
+import sampleBook from '../../sampleData/sampleBook'
+
 
 const App = () => {
   const [allBooks, setAllBooks] = useState(sampleBooks)
 
   return (
     <div className="App">
       <Header />
-      <Books allBooks={allBooks}/>
+      <BookInfo selectedBook={sampleBook}/>
+      {/* <Books allBooks={allBooks}/> */}
```

Comment:
When I set up router, I will create a route with a dynamic URL that matches then ISBN of the book using match.params. This route will pass the selected book as a prop to book info, and render that page.